src/app/shared/layouts/engine-wagon-layout-chart/engine-wagon-layout-chart.component.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { NgModule } from "@angular/core";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { By } from "@angular/platform-browser";
+import { BrowserAnimationsModule } from "@angular/platform-browser/animations";
+import { NgxChartsModule } from "@swimlane/ngx-charts";
+
+import { ChartDirective } from "../../charts/chart.directive";
+import { ChartComponent } from "../../charts/chart/chart.component";
+import { LineChartComponent } from "../../charts/line-chart/line-chart.component";
+import { LayoutComponent } from "../layout/layout.component";
+import { OneByTwoLayoutTableChartComponent } from "./one-by-two-layout-table-chart.component";
```

Comment:
These tests are failing, I think it is because it is in the file for the the unit test for EngineWagonLayoutChartComponent but it is creating and testing OneByTwoLayoutTableChartComponent